apps/web/__e2e__/notes.test.ts

```diff
@@ -37,6 +38,67 @@ test("create a note", async ({ page }) => {
   expect(note).toBeDefined();
 });
 
+test("close a note", async ({ page }) => {
+  const app = new AppModel(page);
+  await app.goto();
+  const notes = await app.goToNotes();
+  const note = await notes.createNote(NOTE);
+
+  expect(await notes.editor.getContent("text")).toBe(NOTE.content);
+  const noteCloseButton = page.locator(
+    getTestId("note-close-button-" + (await note?.getTitle()))
+  );
+  await noteCloseButton.click();
+  expect(await notes.editor.getContent("text")).toBe("");
+});
```

Comment:
What's this? We need to keep the tests related to the `auto saving` functionality. General tests related to app do not belong here.